communication/nats/connection_wrap.go

```diff
@@ -49,16 +46,16 @@ func ParseServerURI(serverURI string) (*url.URL, error) {
 		return nil, errors.Wrapf(err, `failed to parse NATS server URI "%s"`, serverURI)
 	}
 	if serverURL.Port() == "" {
-		serverURL.Host = fmt.Sprintf("%s:%d", serverURL.Host, BrokerPort)
+		serverURL.Host = fmt.Sprintf("%s:%d", serverURL.Host, DefaultBrokerPort)
 	}
 
 	return serverURL, nil
 }
 
-// NewConnection create new ConnectionWrap to given servers`
-func NewConnection(serverURIs ...string) (*ConnectionWrap, error) {
+func newConnection(serverURIs ...string) (*ConnectionWrap, error) {
```

Comment:
It there a point to make this constructor private?

Reply:
Yes, it's not used and not intended to be used outside of the package.